dojo/jira_link/views.py

```diff
@@ -95,6 +95,11 @@ def webhook(request):
             commentor = parsed['comment']['updateAuthor']['displayName']
             jid = parsed['comment']['self'].split('/')[7]
             jissue = JIRA_Issue.objects.get(jira_id=jid)
+            jira = JIRA_Conf.objects.values_list('username', flat=True)
+            for jira_userid in jira:
+                if jira_userid.lower() in commentor.lower():
```

Comment:
This condition will not work in all cases as the `displayName` inside Jira webhooks is not necessarily equal to the user name configured inside DDJ. The `displayName` in my integration is equal to the user + the name (john doe) and inside DDJ is the email address (john.doe@domain).

Reply:
proposal for a more robust mechanism here -> https://github.com/DefectDojo/django-DefectDojo/pull/2815